src/framework/ui/icon/icon.component.tsx

```diff
@@ -27,6 +27,7 @@ export type IconElement<T> = React.ReactElement<T>;
  * `Icon` component with animation support. Allows to render any ReactElement registered for a specific name.
```

Comment:
I believe it will be better to add 'asset-using' examples to all components where it can be.